main.nf

```diff
@@ -261,6 +261,9 @@ if(params.aligner == 'star'){
     if(params.hisat2_index)        summary['HISAT2 Index'] = params.hisat2_index
     else if(params.fasta)          summary['Fasta Ref']    = params.fasta
     if(params.splicesites)         summary['Splice Sites'] = params.splicesites
+} else if(params.transcriptome) {
```

Comment:
```suggestion
} else if(params.aligner == 'salmon') {
```

Reply:
Both of these need to be true for `Salmon` to run.

Another example where it will need to be changed:
https://github.com/nf-core/rnaseq/blob/87185bac003497b4e4fa6a0bedc0ce170d24db42/main.nf#L455

and here:
https://github.com/nf-core/rnaseq/blob/87185bac003497b4e4fa6a0bedc0ce170d24db42/main.nf#L1083